debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorConfig.java

```diff
@@ -540,15 +540,14 @@ public static SchemaRefreshMode parse(String value) {
     public static final Field PLUGIN_NAME = Field.create("plugin.name")
             .withDisplayName("Plugin")
             .withGroup(Field.createGroupEntry(Field.Group.CONNECTION_ADVANCED_REPLICATION, 0))
-            .withEnum(LogicalDecoder.class, LogicalDecoder.DECODERBUFS)
+            .withEnum(LogicalDecoder.class, LogicalDecoder.YBOUTPUT)
```

Comment:
Shouldn't this change be made in a Yugabyte specific connector config class ? Similar to the approach for the hostname config ?

Reply:
I think this should be fine, we just had to change the default value in the defined field.

For the hostname config, we had to override and already existing field but since this plugin name field is only PostgresConnector specific, there's no other place we can change the default value.